src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -25,9 +23,9 @@ public FindCommand parse(String args) throws ParseException {
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
         }
 
-        String[] nameKeywords = trimmedArgs.split("\\s+");
+        //String[] nameKeywords = trimmedArgs.split("\\s+");
```

Comment:
perhaps we can remove this line since we are not using it already

Reply:
resolved!